core/src/main/scala/org/broadinstitute/dsde/rawls/expressions/ExpressionParser.scala

```diff
@@ -12,11 +12,19 @@ import cats.instances.try_._
 
 object ExpressionParser {
   def parseMCExpressions(inputs: Map[String, AttributeString], outputs: Map[String, AttributeString], allowRootEntity: Boolean, parser: SlickExpressionParser): ParsedMCExpressions = {
+    val noEntityAllowedErrorMsg = "Only allowed when running with workspace data model. However, workspace attributes can be used."
```

Comment:
Nitpicky wording for error messages - The error message say _what_ is only allowed when running with the data model. So maybe something along the lines of:

"Expressions beginning with "this." are  only allowed when running with the workspace data model..."